collect_app/src/main/java/org/odk/collect/android/tasks/InstanceUploaderTask.java

```diff
@@ -75,7 +77,7 @@
     private static final int CONNECTION_TIMEOUT = 60000;
     private static final String fail = "Error: ";
     private static final String URL_PATH_SEP = "/";
-
+    private SharedPreferences sharedPreferences;
```

Comment:
Hmm. Could either be that the other one had multiple methods accessing it or it could also just have been a mistake. 😊 Best to keep as tight of a scope as possible, I think.